docs/CHANGELOG/changelog-0.12.x.md

```
@@ -65,3 +65,28 @@ kubectl exec -it <primary-mongo-pod> -n claudie -- mongosh \
 
 ## Bug fixes
 - Fixed deletion of zero sized nodepools that would result in an endless reconciliation loop [#2049](https://github.com/berops/claudie/pull/2049)
+
+
+## v0.12.1
+
+## What's Changed
+- New feature introduced 'upgrade-lock' label. When set on nodes, it signals to Claudie to skip node drain on those nodes, blocking the workflow of pending changes until the label is removed from the nodes. [#2062](https://github.com/berops/claudie/pull/2062)
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)